packages/acceptance-tests/pkg-tests-specs/sources/protocols/git.test.js

```diff
@@ -0,0 +1,144 @@
+const {
+  fs: {readJson, readFile},
+  tests: {getPackageDirectoryPath},
+} = require('pkg-tests-core');
+const {parseSyml} = require('@berry/parsers');
+
+describe(`Protocols`, () => {
+  describe(`git:`, () => {
+    test(
+      `it should resolve a dependency with git tag via git protocol`,
+      makeTemporaryEnv(
+        {
+          dependencies: {[`util-deprecate`]: `git://github.com/TooTallNate/util-deprecate.git#v1.0.1`},
```

Comment:
We heavily base `plugin-git` acceptance tests on `util-deprecate`. Have a few concerns about that:

* what if one day this repository disappears? Maybe we could fork this project? 🤔 
* I can imagine these tests might be flaky due to broken Internet connection. 

WDYT?

Reply:
The best would be to spawn a minimal git server, but I'm not too sure how to do this without a significant investment 🤔

The v1 tests sometime caused timeout, but mostly on large repositories like Yarn itself - `util-deprecate` should be safe, and given that its author is a reasonably known author I'm fairly confident it won't disappear anytime soon.

Reply:
That makes sense, thanks for sharing thoughts 🙇